Update promote tab field information (#6774)
wagtail/wagtail: docs/editor_manual/new_pages/the_promote_tab.rst

The Promote tab
~~~~~~~~~~~~~~~

> The promote tab is where you can configure a page's metadata to help search engines find and index it. Below is a description of all default fields in the promote tab and what they do.

Reviewer: ```suggestion
The Promote tab is where you can configure a page's metadata, to help search engines find and index it. Below is a description of all the default fields under this tab.
```

> * **Slug:** The section of the URL that appears after your website's domain e.g ``http://domain.com/blog/[my-slug]/``. This is automatically generated from the main page title set in the Content tab. Slugs should be entirely lowercase, with words separated by hyphens (-). It is recommended that you don't change a page's slug once a page is published.

Reviewer: ```suggestion
* **Slug:** The section of the URL that appears after your website's domain e.g. ``http://domain.com/blog/[my-slug]/``. This is automatically generated from the main page title, which is set in the Content tab. Slugs should be entirely lowercase, with words separated by hyphens (-). It is recommended that you don't change a page's slug once a page is published.
```

* **Title tag:** This is the bold headline in search engine results. This is one of the most significant elements of how search engines rank the page. The keywords used here should align with the keywords you wish to be found for.

> * **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google chooses to rank your content, but it can impact on the likelihood that a user will click your result. Ideally 140 - 155 characters in length.

Reviewer: ```suggestion
* **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google ranks your content, but it can impact on the likelihood that a user will click your result. Ideally 140 to 155 characters in length.
```

> * **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google chooses to rank your content, but it can impact on the likelihood that a user will click your result. Ideally 140 - 155 characters in length.

Reviewer: Do we want to single out Google here? How about "It has no impact on how most search engines rank your content"?

Reviewer 2: Agreed. Let's not single out Google.

* **Slug:** The section of the URL that appears after your website's domain e.g ``http://domain.com/blog/[my-slug]/``. This is automatically generated from the main page title set in the Content tab. Slugs should be entirely lowercase, with words separated by hyphens (-). It is recommended that you don't change a page's slug once a page is published.
* **Title tag:** This is the bold headline in search engine results. This is one of the most significant elements of how search engines rank the page. The keywords used here should align with the keywords you wish to be found for.
* **Meta description:** This is the descriptive text displayed underneath a headline in search engine results. It is designed to explain what this page is about. It has no impact on how Google chooses to rank your content, but it can impact on the likelihood that a user will click your result. Ideally 140 - 155 characters in length.
* **Show in menus:** Ticking this box will ensure that the page is included in automatically generated menus on your site. Note: A page will only display in menus if all of its parent pages also have *Show in menus* ticked.

> * **Search description:** This field allows you to add text that will be displayed if the page appears in search results. This is especially useful to distinguish between similarly named pages.

Reviewer: Why has this line been removed?

Reviewer 2: @tomdyson This is being renamed to "Meta description"

Reviewer: But in current Wagtail it's `Search description`, right? If we submit a change to this for Wagtail core it should be in the same PR as the code change.
